tests/libvmi/cloudinit.go

```diff
@@ -60,6 +60,17 @@ func WithCloudInitNoCloudNetworkData(data string) Option {
 	}
 }
 
+// WithCloudInitNoCloudEncodedNetworkData adds cloud-init no-cloud base64 encoded network data.
+func WithCloudInitNoCloudEncodedNetworkData(data string) Option {
+	return func(vmi *v1.VirtualMachineInstance) {
+		addDiskVolumeWithCloudInitNoCloud(vmi, cloudInitDiskName, v1.DiskBusVirtio)
+
+		volume := getVolume(vmi, cloudInitDiskName)
+		encodedData := base64.StdEncoding.EncodeToString([]byte(data))
+		volume.CloudInitNoCloud.NetworkDataBase64 = encodedData
```

Comment:
nit: `encodedData` is used only once and could be inlined:
```suggestion
		volume.CloudInitNoCloud.NetworkDataBase64 = base64.StdEncoding.EncodeToString([]byte(data))
```

Reply:
Done